homeassistant/components/keenetic_ndms2/const.py

```diff
@@ -0,0 +1,18 @@
+"""Constants used in the Keenetic NDMS2 components."""
+
+from homeassistant.components.device_tracker.const import (
+    CONF_CONSIDER_HOME as _CONF_CONSIDER_HOME,
```

Comment:
Why are we importing like this and then setting the constants locally again? Either import and use the constant directly or create new constants for this integration.

Reply:
I was avoiding this variable shown unused and dropped by the IDE 
https://user-images.githubusercontent.com/1551926/107527579-83f09200-6beb-11eb-9d6e-9b8d463d5146.mp4

From your options I'll choose to make a copy of the constant